yaptide/celery/utils/pymc.py

```diff
@@ -145,13 +222,11 @@ def average_estimators(base_list: list[dict], list_to_add: list[dict], averaged_
                                if item["metadata"]["page_number"] == page_dict["metadata"]["page_number"]), None)
 
             base_list[est_i]["pages"][page_i]["data"]["values"] = [
-                sum(x) / (averaged_count + 1) for x in zip(
-                    map(lambda x: x * averaged_count, base_list[est_i]["pages"][page_i]["data"]["values"]),
-                    page_dict["data"]["values"]
-                )
+                sum(x) / (averaged_count + 1)
+                for x in zip(map(lambda x: x * averaged_count, base_list[est_i]["pages"][page_i]["data"]["values"]),
+                             page_dict["data"]["values"])
```

Comment:
at some point we should rewrite it to something more human readable, preferably to extract to a function

Reply:
Sure, I separated these functions to have simulator-specific logging but otherwise, they are pretty similar.

Reply:
Updated. Is it ok now?